docs/contributing.md

```diff
@@ -40,7 +40,7 @@ The submitter has the first responsibility of keeping the created PR clean and n
 A PR is required to meet the following criteria.
 * It has passed all the tests defined for TAOS-CI.
     - This includes unit tests and integration tests in various platforms and different static analysis tools.
-    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](CODE_OF_CONDUCT.md). You may need to refer to later section.
+    - Note that one of the tests includes the "Signed-off-by" check, which means that the author has agreed with [Code of Conduct](https://github.com/nnstreamer/nntrainer/blob/main/CODE_OF_CONDUCT.md). You may need to refer to later section.
```

Comment:
Same here. If ```.../CODE_OF_CONDUCT.md``` works, it'd be better.

Reply:
I think just CODE_OF_CONDUCT.md could works. 
(You can see https://github.com/nnstreamer/nntrainer/edit/main/CONTRIBUTING.md#L54 )

Reply:
```markdown
[Code of Conduct](CODE_OF_CONDUCT.md)
```
works in `github.io` but breaks `github.com` 😂
In the [pull request for nnstreamer docs](https://github.com/nnstreamer/nnstreamer/pull/3099), the links were changed to apply the direct link for now.

Maybe I should wait until the [discussion about broken links in the nnstreamer repository](https://github.com/nnstreamer/nnstreamer/issues/3100) ends?

Reply:
If it breaks, then it is ok for this. Thanks.